Context/ContextInterface.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+/*
+ * This file is part of the FOSRestBundle package.
+ *
+ * (c) FriendsOfSymfony <http://friendsofsymfony.github.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FOS\RestBundle\Context;
+
+/**
+ * Stores the serialization or deserialization context (groups, version, ...).
+ *
+ * @author Ener-Getick <egetick@gmail.com>
+ */
+interface ContextInterface
```

Comment:
But if we create 3 interfaces, we have to create 3 context classes ? 
So it's not still standard.

Or do you think it's better to test in each class which context it is ?